src/components/user/IFXUserDetail.vue

```diff
@@ -33,71 +55,95 @@ export default {
 <template>
   <v-container v-if="!isLoading">
     <IFXPageHeader>
-      <template #title>{{item.fullName || id}}<v-icon color="yellow darken-1" class="ml-2 mb-1 key-background" v-if="item.isAdmin">mdi-key</v-icon></template>
+      <template #title>{{item.fullName || id}}</template>
       <template #actions>
         <IFXLoginIcon disabled v-if='item.isActive !== undefined' :isActive='item.isActive'/>
-        <IFXButton btnType="edit" @action="navigateToItemEdit(id)"/>
+        <IFXButton btnType="edit" xSmall @action="navigateToItemEdit(id)"/>
       </template>
       <template #content>
         <IFXItemHistoryDisplay :item='item'/>
       </template>
     </IFXPageHeader>
     <v-container>
-      <v-row no-gutters>
-        <v-col sm="2">
-          <h3>First Name</h3>
-          <h3>Last Name</h3>
-          <h3>Primary Email</h3>
-        </v-col>
-        <v-col sm="3">
-          <div class="data-item">{{item.firstName}}</div>
-          <div class="data-item">{{item.lastName}}</div>
-          <div class="data-item">{{item.primaryEmail}}</div>
-        </v-col>
-        <v-col sm="7">
-          <h3>Groups to which this user belongs</h3>
-          <span v-if="item.groups" class="d-flex flex-column">
-            <div v-for="group in item.groups" :key="group" class="d-flex align-center mt-1">
-              <v-icon class="mr-1">{{$api.group.iconForGroup(group)}}</v-icon>
-              <span>{{group}}</span>
-            </div>
-          </span>
-          <span v-else>None</span>
-        </v-col>
-      </v-row>
-      <v-row>
-      </v-row>
       <v-row>
         <v-col>
-          <IFXItemSelectList
-            title='Contacts'
-            disabled
-            :items='item.contacts'
-            >
-            <template v-slot='{item}'>
-              <IFXSelectableContact :disabled='true' :item='item'/>
-            </template>
-          </IFXItemSelectList>
+          <v-row dense>
+            <v-col sm="3">
+              <h3>First Name</h3>
+            </v-col>
+            <v-col>
+              {{item.firstName}}
+            </v-col>
+          </v-row>
+          <v-row dense>
+            <v-col sm="3">
+              <h3>Last Name</h3>
+            </v-col>
+            <v-col>
+              {{item.lastName}}
+            </v-col>
+          </v-row>
+          <v-row align="start" dense>
+            <v-col sm="3">
+              <h3>Email(s)</h3>
+            </v-col>
+            <v-col>
+              <div class="contact-row">
+                <div class="contact-role">Primary Email</div>
+                <div class="contact-detail"><a :href="`mailto:${item.primaryEmail}`">{{item.primaryEmail}}</a></div>
+              </div>
+                <span v-html="additionalEmailList()"></span>
```

Comment:
`v-html` is kind of frowned upon; I'd suggest moving the code in `additionalEmailList()` here and doing it inline